README.md

```diff
@@ -3,3 +3,49 @@
 Spring conducted a phenotypic, high content imaging screen on inflammasome activated primary human PBMCs using a library of bioactive small molecules to identify novel inflammasome inhibitors. Using images from these phenotypic datasets, we established a computational method to score phenotypic similarity in order to identify compounds that are likely biologically related.
 
 This dataset is designed to benchmark the compound similarity method by allowing measurements of inherent sources of variability such as donors, plates, and experimental runs. We used a bespoke library of 623 compounds, profiled in 4 batches of 8 plates each across a total of 16 donors.
+
+## Explore the data
+
+The data can be visualized and analyzed in the Spring platform at https://app.springscience.com/explore-iinflammasome-benchmark.
+
+## Download the data
+
+The raw image data, along with field metadata and pre-computed features can be accessed at [link coming soon..,].
+The dataset is organized in directories, one for each of the four batches of experiments.
+Each directory includes the following items:
+- The `raw_images` directory contains 2048x2048 16bit TIFF files organized in directories for each plate. The file names encode the location in the plate (`<well>_<field>`, e.g. `A01_f00`), as well as the timepoint `t`, the z-layer `z` and the channel `c` (e.g. `t0z0c0`). Note that for this dataset all images are taken at the same timepoint (`t0`) and z-layer (`z0`).
+- The `immunofluorescence_metadata.json` file provides the information about the immunofluorescence dyes used in the experiment. The order in the `palette` arrays corresponds to `channel` numeral in the image names.
+- The `well_metadata.parquet` file contains the metadata information for each well in the experiment. See below for details about the metadata fields.
+- The `features` directory contains `.parquet` files with pre-computed features for each cell. The files are organized in directories for each plate.
+
+## Metadata
```

Comment:
Only optional, so happy to push back. Do you think we should provide all the unique values for each field? Maybe or maybe not, but it seems like a good high-level view of the data without downloading. Very few people will download such large data and will need to know more about it before committing to the download.